cypress/e2e/michael.cy.ts

```diff
@@ -1,16 +1,18 @@
-describe('drag', () => {
-    xit('click twice', () => {
+describe('obstacle practice', () => {
+    it('click twice', () => {
         cy.visit('https://obstaclecourse.tricentis.com/Obstacles/72954');
-        cy.get('.btn').click().click()
+        cy.contains('a','Click Me 2x').click()
+        cy.contains('a','Click me once more').click()
     });
```

Comment:
Please add cy.contains("Good Job!!") in the end of each test case. Refer to https://github.com/jun1983/cypress_training/blob/main/cypress/e2e/shengjun.cy.ts#L25